# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pacemaker/pacemaker-curses.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -up Pacemaker-1-0-Pacemaker-1.0.5/lib/pengine/Makefile.am.curses Pacemaker-1-0-Pacemaker-1.0.5/lib/pengine/Makefile.am
--- Pacemaker-1-0-Pacemaker-1.0.5/lib/pengine/Makefile.am.curses	2009-08-10 16:42:41.000000000 +0200
+++ Pacemaker-1-0-Pacemaker-1.0.5/lib/pengine/Makefile.am	2011-03-31 09:27:15.964038606 +0200
@@ -34,7 +34,7 @@ libpe_rules_la_SOURCES	= $(rule_files)
 
 libpe_status_la_LDFLAGS	= -version-info 2:0:0
 libpe_status_la_SOURCES	=  $(rule_files) $(status_files)
-libpe_status_la_LIBADD	= -llrm
+libpe_status_la_LIBADD	= -llrm @CURSESLIBS@
 
 clean-generic:
 	rm -f *.log *.debug *~
